components/base/kinematicbase/kinematics.go

```diff
@@ -258,11 +304,17 @@ func (ddk *differentialDriveKinematics) newValidRegionCapsule(starting, desired
 	capsule, err := spatialmath.NewCapsule(
 		center,
 		deviationThreshold,
-		2*deviationThreshold+float64(positionErr),
+		2*deviationThreshold+positionErr,
 		"")
 	if err != nil {
 		return nil, err
 	}
 
 	return capsule, nil
 }
+
+func poseChanged(prevDistErr, prevHeadingErr, distErr, headingErr float64) bool {
```

Comment:
@biotinker, you've worked with metrics the most, which function should we be using here instead of writing something new?

Reply:
This would be a `motionplan.SquaredNormSegmentMetric` with two poses

Reply:
this requires converting an Input array to a Pose every time a new currentInputs is calculated because this function requires having the position in Pose form. is that fine?

Reply:
Actually; I take it back. We should be able to use a `motionplan.L2InputMetric` directly here, not a `SquaredNormSegmentMetric`.

For future reference though; the construction of a `Pose` from a point + orientation as we would be doing here, is not particularly computationally expensive and would be fine to do at the cadence of this code.